### DISCUSSION TITLE
Works for other 7000-series chips as well

### DISCUSSION THREAD

**Discussion by maevii** [isc30/ryzen-gpu-passthrough-proxmox#3] [General]
Thank you so much for this guide! I've been tearing my hair out the past few weeks trying to passthrough the integrated graphics of my 7950X3D, and this is the only thing that's worked so far. I had to extract the vbios manually by compiling the program, but the AMDGopDriver.rom file seemed to work just fine.

I think it'd be a good idea to make this a more general guide for all 7000-series Ryzen chips instead of just one specific system, since a lot of people seem to be having issues passing through their iGPUs (see [this](https://www.reddit.com/r/VFIO/comments/12f2rtw/amd_igpu_passthrough_on_am5_zen4_ryzen_7000_rx570/), [this](https://forum.proxmox.com/threads/ryzen-7000-internal-gpu-passtrough.118910/), [this](https://bbs.archlinux.org/viewtopic.php?id=288505) and [this](https://forums.unraid.net/topic/137695-ryzen-7000-igpu-passthrough-to-vm/)) and this guide seems to work perfectly.

**Comment by isc30** [comment]
wooho! that sounds great!
I also spent an insane amount of time trying to get this working 😄 

I can make a list of "confirmed systems" in the documentation with `7950X3D`, `7735HS` and `7840HS`

**Reply by isc30** [reply]
changed, thanks a lot :)

**Comment by isc30** [comment]
@romner-set hey can you do me a favour and open a PR adding your VBIOS to the repo pls? It could help other people too :)

**Reply by maevii** [reply]
Sure, check #4. Thanks again for the guide!
